app/src/main/kotlin/com/simplemobiletools/dialer/models/PhoneAccountHandleModel.kt

```diff
@@ -0,0 +1,7 @@
+package com.simplemobiletools.dialer.models
+
+data class PhoneAccountHandleModel(
```

Comment:
maybe this needs to be added to proguard exceptions to avoid minimizing in Release mode? Have you tested the fix in Release mode?